integration-tests/src/fees.rs

```diff
@@ -3,35 +3,85 @@
 use crate::kusama_test_net::*;
 use basilisk_runtime::TransactionPayment;
 use frame_support::weights::GetDispatchInfo;
+use primitives::nft::ClassType;
 use sp_runtime::codec::Encode;
 use xcm_emulator::TestExt;
 
+macro_rules! assert_eq_approx {
+	( $x:expr, $y:expr, $z:expr, $r:expr) => {{
+		let diff = if $x >= $y { $x - $y } else { $y - $x };
+		if diff > $z {
+			panic!("\n{} not equal\nleft: {:?}\nright: {:?}\n", $r, $x, $y);
+		}
+	}};
+}
+
+//NOTE: rust encoded call size is differen from UI encoded call size that's why we have asserts for 2 fees.
+
 #[test]
-// #[ignore]
-//TODO: fix this, fees calculcated in this test are apporx 2bsx off.
 fn transfer_transaction_fees() {
-	//This test is not correct
 	Basilisk::execute_with(|| {
+		let diff = UNITS / 100; //0.01
+
+		let expected_rust_encoded_fees = 45_56 * UNITS / 100; //45.56
+		let expected_ui_fees = 4_655 * UNITS / 100; //46.55
```

Comment:
what is the reason for having these comments here, describing the values of the variables?

The thing is that once the UNITS changes, then these comment can easily become outdated.

Reply:
yes, It describe expected fees value in readable form. `UNITS` can't change without this test failing so whoever will be fixing this test can update comments.